main/lv_conf.h

```diff
@@ -0,0 +1,1127 @@
+/**
+ * @file lv_conf.h
+ * Configuration file for v9.2.2
+ */
+
+/*
+ * Copy this file as `lv_conf.h`
+ * 1. simply next to the `lvgl` folder
+ * 2. or any other places and
+ *    - define `LV_CONF_INCLUDE_SIMPLE`
+ *    - add the path as include path
+ */
+
+/* clang-format off */
+#if 1 /*Set it to "1" to enable content*/
+
+#ifndef LV_CONF_H
+#define LV_CONF_H
+
+/*If you need to include anything here, do it inside the `__ASSEMBLY__` guard */
+#if  0 && defined(__ASSEMBLY__)
+#include "my_include.h"
```

Comment:
I think it's safe to comment this out

Reply:
This whole file is a copy of the template supplied by lvgl, with only the changes to defiate from the defaults which minimises the changes. The other option would be to make a tiny lv_conf.h file, which I can take a stab at.

Reply:
Made a minimal `lv_conf.h`.